include/tests_ssh

```diff
@@ -97,6 +97,9 @@
                 IgnoreRhosts:YES,,NO\
                 UseDNS:YES,,NO\
                 X11Forwarding:NO,,YES\
+                FingerprintHash:SHA256,MD5,\
```

Comment:
FingerprintHash:

This one is interesting.  FingerprintHash isn't present in man sshd_config(5) under OpenSSH_6.7p1, which is current in LMDE2.  I know @mboelen is hesitant to include SSH directives which are *too* modern -- because people are likely to break their own systems by applying suggestions that aren't compatible.

Because SHA256 is the default (per http://man7.org/linux/man-pages/man5/sshd_config.5.html), is this really worth including?